src/utils/index.ts

```diff
@@ -124,3 +114,36 @@ export const standConfig = {
 export const contextNotInitialized = (message: string) => () => {
     throw new Error(message);
 };
+
+interface EntityData {
+    id: string | number;
+    name: string;
+}
+
+interface TEntity {
+    id: string | number;
+    name?: string | null;
+    email?: string;
+    [key: string]: unknown;
+}
+
+export function arrayToAppliedString(
```

Comment:
T extends TEntity

array: T[]

key: keyof T

Reply:
This one fix your previous comment, thanks)